modules/gameSearch/src/main/UserGameSearch.scala

```diff
@@ -25,7 +26,8 @@ final class UserGameSearch(
       page = page
     )
 
-  def requestForm(implicit req: Request[_], formBinding: FormBinding) = forms.search.bindFromRequest()
+  def requestForm(implicit req: Request[_], formBinding: FormBinding, lang: Lang) =
```

Comment:
is `lang` required here?

Reply:
It seems so

![implicit](https://user-images.githubusercontent.com/12477220/147647131-3dec89d6-bc69-4d23-934e-9ee7c35c9eca.png)

Reply:
ah yes, you're correct, it is.